documentation/docs/exchanges/bitmart.md

```diff
@@ -0,0 +1,43 @@
+---
+tags:
+- spot exchange connector
+---
+
+# `bitmart`
+
+## 📁 [Connector folder](https://github.com/CoinAlpha/hummingbot/tree/development/hummingbot/connector/exchange/bitmart)
+
+## ℹ️ Exchange Info
+
+**BitMart**
+[Website](https://www.bitmart.com/) | [CoinMarketCap](https://coinmarketcap.com/exchanges/bitmart/) | [CoinGecko](https://www.coingecko.com/en/exchanges/bitmart)
+
+- API docs: https://developer-pro.bitmart.com/en/
+- Transaction fees: https://www.bitmart.com/fee/en
+- Minimum order size: Equivalent value of 5 USDT
+- Creating API keys: https://support.bmx.fund/hc/en-us/articles/360016076854-How-to-Create-An-API
```

Comment:
Ideally, we only source pages that are on the Bitmart website. However, if there's no page in their help center or support, external sources are OK.

Reply:
Copy on this sir. I found a link on their help center regarding fees, I change the link to that one.